docs/api.md

```diff
@@ -8,16 +8,27 @@ Use auro-accordion-group if you want to have auto closing accordion components w
 | Property     | Attribute    | Type      | Default | Description                                      |
 |--------------|--------------|-----------|---------|--------------------------------------------------|
 | `alignRight` | `alignRight` | `Boolean` |         | If set, the trigger content will align right.    |
-| `chevron`    | `chevron`    | `Boolean` |         | If set, the accordion trigger will have a chevron showing expanded/collapsed state. |
+| `emphasis`   | `emphasis`   | `Boolean` |         | If set, emphasis styles will be applied to the auro-accordions. |
 | `expanded`   | `expanded`   | `Boolean` | false   | If set, the accordion is expanded.               |
 | `fluid`      | `fluid`      | `Boolean` |         | If set, the trigger and content will be 100% width. |
+| `group`      | `group`      | `Boolean` |         | Attribute will be set on accordion when it appears in an accordion group. |
+| `iconRight`  | `iconRight`  | `Boolean` |         | If set, the chevron icon will render to the right of the trigger. |
+| `lg`         | `lg`         | `Boolean` |         | If set, the auro-accordion elements will appear larger than normal. |
+| `noChevron`  | `noChevron`  | `Boolean` |         | If set, the chevron icon will not appear inside the trigger of the accordion. |
+| `sm`         | `sm`         | `Boolean` |         | If set, the auro-accordion elements will appear smaller than normal. |
```

Comment:
While I do not disagree with the enumerated attributes, I think there does need to be a further conversation about this and whether we want this work implemented.

1. I can definitely see the case to change the `sm` and `lg` attributes to an enumerated attribute, as it would simplify the API and these features could not overlap in any way.

2. When it comes to the `chevron` attribute, I am not sure I agree that it should be an enumerated attribute. Per the design, the chevron should be present in the accordion by default. I do not believe that a user should need to add an attribute to the component in order to use the default design/component. For our basic version, there should be no attributes and the API should be as simple as possible. I am in favor of keeping the `noChevron` and `iconRight` attributes as they are. We have had multiple discussions about how it is not on the Auro team if a user does not implement and use our component the way we intend, although I see the fact that the way we have these attributes set up, it goes against our guidelines that you have highlighted.